src/modules/navigator/rtl.cpp

```diff
@@ -696,12 +695,13 @@ float RTL::calculate_return_alt_from_cone_half_angle(float cone_half_angle_deg)
 
 	float return_altitude_amsl = _destination.alt + _param_rtl_return_alt.get();
 
+	// If we are close enough, arbitrarily set altitude proportionally to distance to the destination
```

Comment:
I don't feel this comment is necessary, given that it's already explained 5 lines above.